tests/test_writer.py

```diff
@@ -1765,8 +1765,7 @@ def test_viewerpreferences():
     assert v.center_window == True  # noqa: E712
     v.center_window = False
     assert (
-        writer._root_object["/ViewerPreferences"]["/CenterWindow"]
-        == False  # noqa: E712
+        writer.root_object["/ViewerPreferences"]["/CenterWindow"] == False  # noqa: E712
```

Comment:
Is there a specific reason for the `noqa` here? While `is False` might fail because the left side is a `BooleanObject`, using

```suggestion
        writer.root_object["/ViewerPreferences"]["/CenterWindow"] == BooleanObject(False)
```

should still be possible? Or am I mistaken?

Reply:
you are correct `is False` is failing.  `== False` works and the noqa is just here to prevent automatic tools to change the code
`== BooleanObject(False)` might work (not tested yet) but will spend some CPU to build the object and 

I would rather stay with legacy code.
Your opinion?

Reply:
Thanks for the explanation. Yes, I have seen that this is legacy code and in general I like to avoid such `noqa` markers while there actually is a safe workaround. I am going to mark this as resolved as it has been like this before.